app/Views/Templates/components/actions/dropdown/index.blade.php

```diff
@@ -16,8 +14,9 @@
         'primary' => 'btn btn-primary',
         'secondary' => 'btn btn-secondary',
         'accent' => 'btn btn-accent',
-        'ghost' => 'btn btn-tertiary',
+        'ghost' => 'btn btn-tertiary text-white text-xl',
```

Comment:
By hard coding the text color in here we won't be able to do dark mode. effectively. We should rarely have the need to use `text-[color]` instead it should use the emantic color names if needed